Tests/AppTests/GithubTests.swift

```diff
@@ -347,11 +338,10 @@ class GithubTests: AppTestCase {
         // https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/761
         // setup
         Current.githubToken = { "secr3t" }
-        let pkg = Package(url: "https://github.com/foo/bar")
         let client = MockClient { _, resp in resp.status = .notFound }
 
         // MUT
-        let res = await Github.fetchLicense(client: client, packageUrl: pkg.url)
+        let res = await Github.fetchLicense(client: client, owner: "foo", repository: "bar")
 
         // validate
         XCTAssertEqual(res, nil)
```

Comment:
We have a warning on line 456 (see below):

> Initialization of immutable value 'images' was never used; consider replacing with assignment to '_' or removing it

It's a bit weird, it doesn't show up consistently in Xcode for me for some reason, so it's easy to miss.

One way of fixing it would be to just check on `images`:

```swift
XCTAssertEqual(images, [])
```

Reply:
Odd that it had not shown up in Xcode for me at all!

Fixed